### ISSUE TITLE
How to install Arrow

### ISSUE THREAD

**Issue by KnudNorNielsen** [marbl/canu#723]
I apologize if this issue should have been posted elsewhere, but it was the best place I could find!
  
I have with great success assembled a 40 Mb fungal genome using CANU (Thanks for a good program).

Now I which to polish the assembly with Arrow, but I have had a hard time figuring out how to. 

As far as I understand I should download SMRT Link v5.0.1 (http://www.pacb.com/support/software-downloads/).
- Do I need a licence, or is this free?  
- Should it be placed in the same directory as the input files?
- When the SMRT Link download have been unpacked, can I then just run the code given at https://github.com/PacificBiosciences/GenomicConsensus, or is there some installation, or configuration that need to been done?

Basically, I simply don't get what I am meant to do in order to get Arrow running. Could anyone please help me out please.

When, should I consider running ArrowGrid instead of Arrow?
 
I sincerely hope someone can shed light on this for me, so I can publish my super assembled genome :-)

**Comment by skoren** [comment]
Yes, you need SMRTlink and it should be available to anyone. You need to install following their directions and make sure the binaries are in your path and runnable (like bax2bam, arrow, dataset, variantCaller). GenomicConsensus is already part of the SMRTlink install.

Arrow itself is consensus algorithm but it requires several steps to run such as alignment, asm partitioning, and finally consensus. The easiest way is to use the ArrowGrid wrapper (https://github.com/skoren/ArrowGrid) which runs the above pipeline for you but it still requires SMRTlink to be installed.
